core/src/v2/schema.rs

```diff
@@ -236,7 +244,16 @@ impl<T: Apiv2Schema> Apiv2Schema for Option<T> {
     }
 }
 
-impl<T, E> Apiv2Schema for Result<T, E> {}
+impl<T, E> Apiv2Schema for Result<T, E> {
+    default const NAME: Option<&'static str> = None;
+
+    /// Returns the raw schema for this object.
```

Comment:
Can you also remove the doc comment here (and above)? IDEs and docs pages generally default to the documentation in trait methods. We should add doc comments here only if we're doing something different from what the actual doc comment says.